lib/program-helper.js

```diff
@@ -79,8 +83,6 @@ Helper.getTSCCommand = function () {
 
 /**
  * Resolve ts file paths
- *
- * @returns {Array}
```

Comment:
+1 for not using semi-useless "Array" type here

Here is what I would do:

```suggestion
 *
 * @returns {string[]}
```